doc/adr/0013-synchronous_peer_discovery.md

```diff
@@ -0,0 +1,82 @@
+# 13. Synchronous Peer Discovery
+
+Date: 2023-04-19
+
+## Status
+
+Accepted
+
+## Context
+
+In order to increase security, it is desirable to observe only peers that are on the majority fork.
+This can be determined by querying discovered peers for their latest ordinal values prior to
+selecting one to observe. Afterwards, the majority ordinal can be determined and each peer is queried
+again for the `Signed[GlobalSnapshot]` corresponding to that ordinal. The hashes of the snapshots
```

Comment:
Maybe we can add an endpoint asking for the hash of the snapshot at a given ordinal to not download a snapshot from every node just to determine majority?

Reply:
Yeah, @cngo-github you should make a separate ticket for this to optimize this, I think this came up earlier but current implementation is fine for now.

Reply:
Ticket created.